phoenix-core/src/it/java/org/apache/phoenix/end2end/UpsertSelectIT.java

```diff
@@ -252,158 +291,164 @@ public void testUpsertSelectEmptyPKColumn() throws Exception {
-        assertEquals(null, rs.getBigDecimal(4));
-        
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW7, rs.getString(2));
-        assertTrue(rs.getDate(3).before(now) );
-        assertTrue(BigDecimal.valueOf(7).compareTo(rs.getBigDecimal(4)) == 0);
-        
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW8, rs.getString(2));
-        assertTrue(rs.getDate(3).before(now) );
-        assertTrue(BigDecimal.valueOf(6).compareTo(rs.getBigDecimal(4)) == 0);
-        
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW9, rs.getString(2));
-        assertTrue(rs.getDate(3).before(now) );
-        assertTrue(BigDecimal.valueOf(5).compareTo(rs.getBigDecimal(4)) == 0);
-
-        assertFalse(rs.next());
-        conn.close();
-        
-        conn = DriverManager.getConnection(getUrl(), props);
-        conn.setAutoCommit(true);
-        upsert = "UPSERT INTO " + ptsdbTable + "(\"DATE\", val, inst) " +
-            "SELECT \"DATE\"+1, val*10, host FROM " + ptsdbTable;
-        upsertStmt = conn.prepareStatement(upsert);
-        rowsInserted = upsertStmt.executeUpdate();
-        assertEquals(4, rowsInserted);
-        conn.commit();
-        conn.close();
-        
         Date then = new Date(now.getTime() + QueryConstants.MILLIS_IN_DAY);
         query = "SELECT host,inst, \"DATE\",val FROM " + ptsdbTable + " where inst is not null";
-        conn = DriverManager.getConnection(getUrl(), props);
-        statement = conn.prepareStatement(query);
-        
-        rs = statement.executeQuery();
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW6, rs.getString(2));
-        assertTrue(rs.getDate(3).after(now) && rs.getDate(3).before(then));
-        assertEquals(null, rs.getBigDecimal(4));
-        
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW7, rs.getString(2));
-        assertTrue(rs.getDate(3).after(now) && rs.getDate(3).before(then));
-        assertTrue(BigDecimal.valueOf(70).compareTo(rs.getBigDecimal(4)) == 0);
-        
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW8, rs.getString(2));
-        assertTrue(rs.getDate(3).after(now) && rs.getDate(3).before(then));
-        assertTrue(BigDecimal.valueOf(60).compareTo(rs.getBigDecimal(4)) == 0);
-        
-        assertTrue (rs.next());
-        assertEquals(null, rs.getString(1));
-        assertEquals(ROW9, rs.getString(2));
-        assertTrue(rs.getDate(3).after(now) && rs.getDate(3).before(then));
-        assertTrue(BigDecimal.valueOf(50).compareTo(rs.getBigDecimal(4)) == 0);
-        
-        assertFalse(rs.next());
-        conn.close();
-        
+
+        try (Connection conn = DriverManager.getConnection(getUrl(), props);
```

Comment:
Here too, spaces added because of additional indentation because of try-with-resources.